.github/workflows/jules-workflows.yml

```
@@ -459,10 +425,8 @@ jobs:
     - name: Configure Git
       uses: ./.github/actions/configure-git
     - name: Run implementers sequentially
-      id: run-implementers
-      env:
-        IMPLEMENTER_MATRIX: ${{ needs.generate-planner-matrix.outputs.implementer_matrix }}
-      run: bash .github/scripts/jules-run-implementers-sequential.sh
+      id: run
+      run: jlo workflow run implementers --matrix-json '${{ fromJSON(needs.generate-planner-matrix.outputs.json).implementer_matrix | toJSON }}' --target-branch '${{ env.TARGET_BRANCH }}' $${{ env.JLO_RUN_FLAGS }}
```

actionlint (1.7.10)

[error] got unexpected character ' ' while lexing || operator, expecting '|'

(expression)